tests/bal_functional_behaviour_suite.py

```diff
@@ -0,0 +1,169 @@
+#
+#   Copyright 2013-2023 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+A manager test harness test case suite that validates that the
+BasicAssetLibrary manager behaves as expected concerning functional
+behaviors that don't fall under "business-logic", such as performance
+related requirements.
+"""
+
+# pylint: disable=invalid-name, missing-function-docstring, missing-class-docstring
+
+import os
+from unittest import mock
+
+from openassetio import Context, TraitsData
+from openassetio.test.manager.harness import FixtureAugmentedTestCase
+
+
+class Test_simulated_latency_default(FixtureAugmentedTestCase):
+    shareManager = False
```

Comment:
[fix] The doc strings for the class should go before any vars etc... which makes it a little clearer how its working too.

Reply:
5015d7da77acf060943f97273ee912ffa300e063